packages/overlay/src/Overlay.ts

```diff
@@ -534,6 +540,9 @@ export class Overlay extends OverlayFeatures {
     }
 
     protected override requestSlottable(): void {
+        if (this.lastRequestSlottableState === this.open) {
+            return;
+        }
```

Comment:
Great catch on the stutter, @najikahalsema, we should use broken browsers more often? This will prevent duplicate `slottable-request` events from being dispatched with the same `data`, which appears to be what was causing what you captured, even if it was really difficult to run into normally.